### PR TITLE
Added .env to .gitignore

### PR DESCRIPTION
**Pull request by ellbosch**
Added .env to .gitignre. Cleared local cache to reset ignored files which removed multiple files in .vscode folder.

### CODE REVIEW

**Comment by ellbosch** [comment]
@pensivebrian thoughts on this? Some pros/cons to removing the .env file, we'd probably need to update our build instructions so that developers are told to add their own .env file.

**Comment by ellbosch** [comment]
Removed .vscode from .gitignore so that launch and settings files could be included. However, noticed some issues with changes to the files. Will re-assess and make another commit.

**Comment by pensivebrian** [comment]
Ideally, we'd leave .vscode in the .gitignore so that changes to the launch and settings .json files don't show up in git. At the same time, we check them in so that the files are seeded when you run a git clone (I hope this is how is works). This does mean when we want to update one of these .json files, we have to explicitly force the change via the -f option.

File: .env [hunk 1]
```diff
@@ -1,4 +0,0 @@
-MSSQL_CLI_SERVER=
```
Comment (pensivebrian) [inline .env]: We still want this file to be checked in so that devs know what variables to set.

File: .gitignore [hunk 2]
```diff
@@ -112,8 +112,8 @@ env*/
 # Pycharm
 .idea/
 
-# vscode
-.vscode/
```
Comment (pensivebrian) [inline .gitignore]: I don't think we want to delete this.

Comment (ellbosch) [reply .gitignore]: Right, I'll go fix that. Thanks Brian.

**Pull request merged**